core/file/atime_spec.rb

```diff
@@ -15,6 +15,17 @@
     File.atime(@file).should be_kind_of(Time)
   end
 
+  platform_is :linux do
+    it "returns the last access time for the named file with microseconds" do
+      3.times do
```

Comment:
> The cost of doing this repeatedly is pretty low, so this might as well be 10 or 100.

Increased to 100.

> The alternative would be to create a temporary file and forcibly modify these times, but I don't know if that's even possible for atime.

That's possible for `atime` and `mtime` via [FileUtils#touch](http://ruby-doc.org/stdlib-2.4.0/libdoc/fileutils/rdoc/FileUtils.html#method-c-touch).

This alternative was used.

> Note also that some Linux systems disable atime (e.g. via mount params) for better filesystem speed. I don't know if that will ever affect specs, though.

This comment added to code, if you don't mind.